### PR TITLE
Create ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-53381](https://cumulocity.atlassian.net/browse/MTM-53381)
Original PR: [5646](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5646)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix UI Error when Editing Smart Rule from Application Context
```

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-53381
+version: 1019.6.11
+---
+UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:17]: The Claude AI suggested to use the following body:
```suggestion
Fixed a UI error that occurred when editing a smart rule from the application context version 2. This prevents the UI from crashing when making edits to smart rules.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by janhommes** [comment]
@pmich-sag can you please check the suggestions and maybe directly correct them? For me it is totally not clear what "application context v2" is.

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md (hunk 1 above)
Comment (pmich-sag) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:3]:
```suggestion
title: Fix UI error when editing Smart rule from Application context
```

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (pmich-sag) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:17]:
```suggestion
Fixed an UI error that occurred when editing a Smart rule from the Application context. This prevents sending request with wrong MO id and then displaying alert message in UI when making edits to Smart rules.
```

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md (hunk 2 above)
Comment (pmich-sag) [reply content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:17]: Just as above. Change this suggestion a little.

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-53381
+version: 1019.6.11
+---
+Fixed an UI error that occurred when editing a Smart rule from the Application context. This prevents sending request with wrong MO id and then displaying alert message in UI when making edits to Smart rules.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:17]:
```suggestion
Fixed an UI error that occurred when editing a smart rule from the application context. This fix prevents sending a request with a wrong managed object ID and then displaying an alert message.
```

File: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix UI error when editing Smart rule from Application context
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md:3]:
```suggestion
title: Editing smart rules from the application context no longer displays alert message
```

**Review by pmich-sag — APPROVED** [review]

**Comment by pmich-sag** [comment]
> @pmich-sag can you please check the suggestions and maybe directly correct them? For me it is totally not clear what "application context v2" is.

Done.

**Review by Airdroplet — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**